### ISSUE TITLE
App won't display messages in the guestbook

### ISSUE THREAD

**Issue by barisbasar1209** [flutter/flutter#119429]
Hi, 

As soon as I finished step 7, unfortunately I came to realize that my app won't display the messages stored in the guestbook.
1. I checked the code multiple times and I couldn't find any difference between what I run and what is given in the codelab
2. The messages I write into the guestbook are stored in firestore
3. I tried switching accounts
4. I completed step 8 in order to make sure that the problem is not originating from the setting of the rules

At this point I am out of ideas on what to do or try in order to fix it. I thought maybe this could stem from an error in the code. But honestly I couldn't figure that out if there was one. 
Help would be highly appreciated!
Thanks in advance

Edit : 
I just realized the different steps are literally the code for each step. I fetched step 7 and connected it to firestore. Now with this code it works. Must've been my fault. I'll search for what went wrong, now that I have a reference. Thanks !

**Comment by Paguiar735** [comment]
I'm glad to hear that!

Feel free to share with us the link to both repositories: one with your original project and the one with the fix, so we can try helping spot what went wrong if you feel like it. 🙂

For the time being, do you mind closing the issue?

**Comment by barisbasar1209** [comment]
> I'm glad to hear that!
> 
> Feel free to share with us the link to both repositories: one with your original project and the one with the fix, so we can try helping spot what went wrong if you feel like it. 🙂
> 
> For the time being, do you mind closing the issue?

Thank you ! 
The problem was just that I messed something up with the message query to the database. I wrote message: document.data()['message'] as String. Instead of 'text' , which I thought would make no difference, because I didn't really understand what this part of the code was supposed to do. So all fine with your code :). 
And sure yeah I will close this issue right now. 
Thank you very much for the quick support !!
